data/wanderers start.txt

```diff
@@ -364,6 +364,8 @@ mission "Tell Wanderers about the Pug"
 			choice
 				`	"But wouldn't you rather be free from their control, free to remain on these worlds you have worked so hard to restore?"`
 				`	"If your species is being secretly controlled by another, doesn't that bother you?"`
+				`	"I will try not to anger them."`
```

Comment:
This choice doesn't sound like a response to what Rek just asked. Perhaps instead of a choice that simply ends the mission there, the player is able to respond with something along the lines of "I guess you're right," followed by a response from Rek that then ends the mission.